scripts/ci-deploy.sh

```diff
@@ -21,29 +22,48 @@ if [[ -z ${ANSIBLE_FOLDERS} ]]; then
   ANSIBLE_FOLDERS="infrastructure"
 fi
 
+if [[ -d ${cdk_app_dir} ]]; then
+  INFRASTRUCTURE_FOLDERS=${cdk_app_dir}
+elif [[ -z ${INFRASTRUCTURE_FOLDERS} ]]; then
+  INFRASTRUCTURE_FOLDERS=${ANSIBLE_FOLDERS}
+fi
+
 # shellcheck source=.
 . "$dir"/ci-include.sh
 
-pip --quiet --disable-pip-version-check --no-color install ansible boto3 requests pyyaml awscli netaddr
-
-# install the collection for CI/CD
-ansible-galaxy collection install --force git+https://github.com/ringier-data/ops-ci-aws.git,main
-
-MODULES=("${ANSIBLE_FOLDERS//,/ }")
+MODULES=("${INFRASTRUCTURE_FOLDERS//,/ }")
 
 for module in "${MODULES[@]}"; do
   echo Deploying "${module}" module...
   pushd "${module}"
 
-  if [[ -f "requirements.txt" ]]; then
-    pip install -r requirements.txt
-  fi
+  if [[ -f "./package.json" && -f "./package-lock.json" ]]; then
+    echo "CDK app detected..."
 
-  if [[ -f "playbook.yml" ]]; then
-    ansible-playbook -e env="$ENV" -v playbook.yml
+    echo "Installing dependencies..."
+    npm --no-color ci
+
+    npm --no-color run cdk diff -- --context env=${ENV}
+
+    echo "Deploying to env=${ENV}..."
```

Comment:
Suggest talking it out, see the comment above.

Reply:
Done